klient/wait/wait_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package wait
+
+import (
+	"context"
+	"log"
+	"testing"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+
+	"sigs.k8s.io/e2e-framework/klient/k8s"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+)
+
+func TestPodRunning(t *testing.T) {
+	var err error
+	pod := createPod("p1", t)
+	err = For(conditions.New(getResourceManager()).WithVerboseLog().PodRunning(pod), WithImmediate())
```

Comment:
I don't think it's a good idea to control logging this way.
Log control should be independent of that the `wait` package
and should be done universally the same way.

Reply:
xref: https://github.com/kubernetes-sigs/e2e-framework/pull/73#discussion_r752862420